### ISSUE TITLE
getprimary command is wrong

### ISSUE THREAD

**Issue by lkthomas** [ansible-collections/ansible.mysql#571]
Code on file "plugins/modules/mysql_replication.py" seems using the wrong command, 

line 27 of that file shown below,
```module: mysql_replication
short_description: Manage MySQL replication
description:
- Manages MySQL server replication, replica, primary status, get and change primary host.
author:
- Balazs Pocze (@banyek)
- Andrew Klychkov (@Andersson007)
options:
  mode:
    description:
    - Module operating mode. Could be
      C(changeprimary) (CHANGE PRIMARY TO),
      C(getprimary) (SHOW PRIMARY STATUS),
```

it should be "SHOW MASTER STATUS" instead of "SHOW PRIMARY STATUS"

mysql> show master status\G
*************************** 1. row ***************************
             File: mysql-bin.000002
         Position: 157
     Binlog_Do_DB: 
 Binlog_Ignore_DB: 
Executed_Gtid_Set: 
1 row in set (0.00 sec)

mysql> show primary status\G
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'primary status' at line 1
mysql> show variables like '%version%';
+--------------------------+------------------------------+
| Variable_name            | Value                        |
+--------------------------+------------------------------+
| admin_tls_version        | TLSv1.2,TLSv1.3              |
| immediate_server_version | 999999                       |
| innodb_version           | 8.1.0                        |
| original_server_version  | 999999                       |
| protocol_version         | 10                           |
| replica_type_conversions |                              |
| slave_type_conversions   |                              |
| tls_version              | TLSv1.2,TLSv1.3              |
| version                  | 8.1.0                        |
| version_comment          | MySQL Community Server - GPL |
| version_compile_machine  | x86_64                       |
| version_compile_os       | Linux                        |
| version_compile_zlib     | 1.2.13                       |
+--------------------------+------------------------------+
13 rows in set (0.02 sec)

**Comment by laurent-indermuehle** [comment]
Hi @lkthomas and thank you for taking the time to open this issue.
Did I read correctly and you are using the new MySQL 8.1? If so, we haven't tested it yet with this collection.
Did Oracle have undone the change "master->primary" and "slave->replica" in MySQL 8.1?
Have you tested with MySQL 8.0?

The container for 8.1 is available since two months. So it should not be too hard to add this version to our integration tests. @lkthomas, have you some time for this? I'll be happy to assist you. It's just I don't have time on my own until next year... at best.

**Comment by lkthomas** [comment]
https://dev.mysql.com/doc/refman/8.0/en/show-master-status.html
https://dev.mysql.com/doc/refman/8.1/en/show-master-status.html
https://dev.mysql.com/doc/refman/5.7/en/show-master-status.html

All official documentation stated it's using "SHOW MASTER STATUS", even from version 5.7

Are you sure you are testing using the official version and not a custom-patched version? wording could change because of third party patches. 

Correct I am using MySQL 8.1 official stable binary.

**Comment by lkthomas** [comment]
https://mariadb.com/kb/en/show-binlog-status/

SHOW MASTER STATUS was renamed to SHOW BINLOG STATUS in [MariaDB 10.5.2](https://mariadb.com/kb/en/mariadb-1052-release-notes/), but the old name remains an alias for compatibility purposes.

I think it's wrong, let me see if I could make a change

**Comment by laurent-indermuehle** [comment]
Thanks @lkthomas for taking the time to open a PR. But first I want to clarify the situation as I'm not sure ours modules cause any issue.

To me, it was clear that both MySQL and MariaDB will stop using the words "Master" and "Slave".

As it turns out, Oracle decided to change Master to "Source" in 2020: https://dev.mysql.com/blog-archive/mysql-terminology-updates/ Thought I never encounter that word anywhere yet and I prefer "primary".

But I think like you that the document should be align with the code. The error message you posted was from MySQL directly. But did the Ansible Module cause you an yissue? Maybe mysql_replication with the getprimary option? I think we are aliasing Master<->Primary and it will work in all cases.

**Comment by lkthomas** [comment]
No issue to use the module, it just at the first glance from the document it seems wrong, that's why I did a PR so the document will get changed; If it's an aliasing, shouldn't it be stating on the document as well?

**Comment by laurent-indermuehle** [comment]
Right @lkthomas, I get it now. In parenthesis we wrote the equivalent SQL query that is indeed wrong. I just tested MariaDB 10.6, `SHOW PRIMARY STATUS` don't work.
The `CHANGE PRIMARY TO` is also wrong.
I'll have a second look on your PR.

**Comment by laurent-indermuehle** [comment]
Closed by #573
